akka-remote/src/main/scala/akka/remote/artery/aeron/ArteryAeronUdpTransport.scala

```diff
@@ -95,6 +95,7 @@ private[remote] class ArteryAeronUdpTransport(_system: ExtendedActorSystem, _pro
         driverContext.aeronDirectoryName(randomName)
       }
       driverContext.clientLivenessTimeoutNs(settings.Advanced.Aeron.ClientLivenessTimeout.toNanos)
+      driverContext.publicationUnblockTimeoutNs(settings.Advanced.Aeron.PublicationUnblockTimeout.toNanos)
```

Comment:
should probably be in `SharedMediaDriverSupport` also

Reply:
Yep, adding now